data-processing-lib/src/data_processing/transform/execution_configuration.py

```diff
@@ -0,0 +1,83 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import argparse
+import ast
+
+from data_processing.utils import CLIArgumentProvider, ParamsUtils, get_logger
+
+
+logger = get_logger(__name__)
+
+
+cli_prefix = "runtime_"
+
+
+class TransformExecutionConfiguration(CLIArgumentProvider):
```

Comment:
ouch a new class. I guess this needs some doc in the doc directory.

Reply:
Yes, but without it it will be a lot of duplication of code